src/components/books/Book.js

```diff
@@ -1,12 +1,21 @@
 /*eslint-disable */
```

Comment:
- [ ] Kindly avoid disabling linters as they help us follow best practices and standards.